website/project/model.py

```diff
@@ -204,7 +230,9 @@ class Comment(GuidStoredObject, SpamMixin, Commentable):
     # The type of root_target: node/files
     page = fields.StringField()
     content = fields.StringField(required=True,
-                                 validate=[MaxLengthValidator(settings.COMMENT_MAXLENGTH), validators.string_required])
+                                 validate=[validators.comment_maxlength(settings.COMMENT_MAXLENGTH), validators.string_required])
+    # The mentioned users
+    ever_mentioned = fields.ListField(fields.StringField())
```

Comment:
"ever_mentioned" is to make it clear that if users have previously been mentioned in a comment they will not be notified for any future mentions in that comment. "mentioned" could be interpreted as the mentions that are added in a particular save of a comment.